dataclients/kubernetes/ingress.go

```diff
@@ -71,20 +72,21 @@ func (ic *ingressContext) addHostRoute(host string, route *eskip.Route) {
 
 func newIngress(o Options) *ingress {
 	return &ingress{
-		provideHTTPSRedirect:     o.ProvideHTTPSRedirect,
-		httpsRedirectCode:        o.HTTPSRedirectCode,
-		pathMode:                 o.PathMode,
-		kubernetesEnableEastWest: o.KubernetesEnableEastWest,
-		kubernetesEastWestDomain: o.KubernetesEastWestDomain,
-		eastWestRangeDomains:     o.KubernetesEastWestRangeDomains,
-		eastWestRangePredicates:  o.KubernetesEastWestRangePredicates,
-		allowedExternalNames:     o.AllowedExternalNames,
-		forceKubernetesService:   o.ForceKubernetesService,
-		backendTrafficAlgorithm:  o.BackendTrafficAlgorithm,
+		provideHTTPSRedirect:         o.ProvideHTTPSRedirect,
+		httpsRedirectCode:            o.HTTPSRedirectCode,
+		pathMode:                     o.PathMode,
+		kubernetesEnableEastWest:     o.KubernetesEnableEastWest,
+		kubernetesEastWestDomain:     o.KubernetesEastWestDomain,
+		eastWestRangeDomains:         o.KubernetesEastWestRangeDomains,
+		eastWestRangePredicates:      o.KubernetesEastWestRangePredicates,
+		allowedExternalNames:         o.AllowedExternalNames,
+		forceKubernetesService:       o.ForceKubernetesService,
+		backendTrafficAlgorithm:      o.BackendTrafficAlgorithm,
+		defaultLoadBalancerAlgorithm: o.DefaultLoadBalancerAlgorithm,
 	}
 }
 
-func getLoadBalancerAlgorithm(m *definitions.Metadata) string {
+func getLoadBalancerAlgorithm(m *definitions.Metadata, defaultLoadBalancerAlgorithm string) string {
```

Comment:
the name seems to be a bit long `defaultLoadBalancerAlgorithm`, maybe `defaultAlgorithm`. The func says already something about load balancer.